udata/core/spatial/api.py

```diff
@@ -32,6 +35,35 @@
     'size', type=int, help='The amount of suggestion to fetch',
     location='args', default=10)
 
+dataset_parser = api.parser()
+dataset_parser.add_argument(
+    'dynamic', type=bool, help='Append dynamic datasets',
```

Comment:
OK, I will improve this documentation to mention input parsers :+1: 
The boolean input is here: http://flask-restplus.readthedocs.io/en/stable/api.html#flask_restplus.inputs.boolean